xml/ha_storage_protection.xml

```diff
@@ -737,11 +740,10 @@ Timeout (msgwait)  : 40
        <literal>sg_persist</literal>:
       </para>
 <screen>&prompt.root;<command>crm</command> configure
-&prompt.crm.conf;<command>primitive</command> sg ocf:heartbeat:sg_persist \
-    params config_file=/etc/sg_persist.conf \
-           sg_persist_resource=MDRAID1 \
-           reservation_type=1 \
-    op monitor interval=60 timeout=60</screen>
+&prompt.crm.conf;<command>primitive</command> sg sg_persist \
+    params devs="/dev/sdb" reservation_type=3 \
+    op start timeout=30 interval=0 op stop timeout=30 \
+    interval=0 op monitor timeout=20 interval=29 role=Slave</screen>
```

Comment:
If we define role=Slave, then better to define role=Master too. Otherwise, to remove "role=" is reasonable to me.

Reply:
>a) s/sdb/sdc/ to make it consistent with "sdc1" in the later paragraph.
done

>b) If we define role=Slave, then better to define role=Master too. Otherwise, to >remove "role=", stay the original "op monitor interval=60 timeout=60" is reasonable to >me.
I decided to remove "role=" and go back to the original "op monitor interval=60 timeout=60". Many thanks!